odoo/addons/base/models/ir_mail_server.py

```diff
@@ -296,6 +308,11 @@ def connect(self, host=None, port=None, user=None, password=None, encryption=Non
         # Anyway, as it may have been sent by login(), all subsequent usages should consider this command as sent.
         connection.ehlo_or_helo_if_needed()
 
+        # Store the "from_filter" of the mail server / odoo-bin argument to  know if we
+        # need to change the FROM headers or not when we will prepare the mail message
+        connection.from_filter = from_filter
```

Comment:
Are those "existing" parameters or are you adding it  as custom on connection ?

Reply:
It's custom attributes